architecture/src/androidLibMain/kotlin/observable/Observable.kt

```diff
@@ -161,7 +162,7 @@ class ObservablePropertySubject<T>(observableProperty: ObservableProperty<T>, co
  * @param flowable The [HotFlowable] to synchronize to
  * @param coroutineScope The [CoroutineScope] on which to observe changes to the [HotFlowable]
  */
-class FlowSubject<T>(private val flowable: HotFlowable<T>, private val coroutineScope: CoroutineScope) : Subject<T>(coroutineScope) {
+class FlowableSubject<T>(private val flowable: HotFlowable<T>, private val coroutineScope: CoroutineScope) : Subject<T>(coroutineScope) {
```

Comment:
I have a very simple local change that adds the `Flow` interface to `Flowable` (it just forwards the collect).

Reply:
By this I mean, maybe we should not rename it, since I think it can easily be made to work for all Flows.

Reply:
Upon further review, this makes perfect sense for now.